pmd-typescript/src/main/java/net/sourceforge/pmd/lang/typescript/ast/TypeScriptLexerBase.java

```diff
@@ -0,0 +1,166 @@
+package net.sourceforge.pmd.lang.typescript.ast;
+
+// Source: https://github.com/antlr/grammarsv4/
+
+import org.antlr.v4.runtime.*;
+
+import java.util.Stack;
+
+/**
+ * All lexer methods that used in grammar (IsStrictMode)
+ * should start with Upper Case Char similar to Lexer rules.
```

Comment:
> TypescriptLexerBase, coming directly from antlr4 repository, doesn't pass checkstyle. Please advise what would be the best approach.

Can you try renaming the methods to use regular camel case? AFAIK this casing convention is not necessary but just an internal convention the authors used in their codebase. Also just format the file, there are some obvious style inconsistencies that checkstyle rightfully reports on (brace placement eg).